### DISCUSSION TITLE
how to disable auto updates?

### DISCUSSION THREAD

**Discussion by iselestia** [bmaltais/kohya_ss#2125] [Q&A] [answered]
im tired from author updates nothing works every time he update kohya, and need reinstall all over to get it work. todays update the same i started train on old worked script, but get me error instead model file. waste of time on this forced updates.

**Comment by dmaltsevQA** [comment, marked as answer]
what file do you run? for example - I have webui-lora-only.cmd and there is "git pull origin master" string - if I delete it, your question decided

**Reply by iselestia** [reply]
eh. If only I had this information a year earlier. there would be no need to make dozens of reinstallations per year. I used to use the portable version before... because the original was installed with errors and did not work. Now I installed the original. albeit with errors. exactly the same after any updates. the program cannot install anything and kohya stops working altogether. but I already manually added all the files that he could not install automatically. in my case it is the sd-scripts folder. but thank you. it really helped.

**Comment by dmaltsevQA** [comment]
I used this installer https://github.com/serpotapov/Kohya_ss-GUI-LoRA-Portable and haven't any problem with updates
